02-Appetisers/Sea_Ice_Coordinates.ipynb

```diff
@@ -54,14 +54,24 @@
     "tags": []
```

Comment:
<p>The <code>decode_coords</code> assignment in this comment looks wrong. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>